CITATION.cff

```diff
@@ -0,0 +1,125 @@
+  climate and forecast data are often not simply representative of points in space/time,
+  other extensions provide for the description of coordinate intervals, multidimensional
+  cells and climatological time coordinates, and indicate how a data value is representative
+  of an interval or cell. This standard also relaxes the COARDS constraints on dimension
+  order and specifies methods for reducing the size of datasets.
+authors:
+- affiliation: NCAR
+  family-names: Eaton
+  given-names: Brian
+- affiliation: University of Reading and UK Met Office Hadley Centre
+  family-names: Gregory
+  given-names: Jonathan
+  orcid: 0000-0003-1296-8644
+- affiliation: PCMDI, LLNL
+  family-names: Drach
+  given-names: Bob
+- affiliation: PCMDI, LLNL
+  family-names: Taylor
+  given-names: Karl
+  orcid: 0000-0002-6491-2135
+- affiliation: PMEL, NOAA
+  family-names: Hankin
+  given-names: Steve
+- affiliation: University of Reading
+  family-names: Blower
+  given-names: Jon
+- affiliation: Unidata
+  family-names: Caron
+  given-names: John
+- affiliation: USGS
+  family-names: Signell
+  given-names: Rich
+- affiliation: UK Met Office
+  family-names: Bentley
+  given-names: Phil
+- affiliation: MIT
+  family-names: Rappa
+  given-names: Greg
+- affiliation: DKRZ
+  family-names: "H\xF6ck"
+  given-names: Heinke
+  orcid: 0000-0002-0131-1404
+- affiliation: NCAS and CEDA
+  family-names: Pamment
+  given-names: Alison
+  orcid: 0000-0001-5040-4626
+- affiliation: NCAS
+  family-names: Juckes
+  given-names: Martin
+- affiliation: SMHI
+  family-names: Raspaud
+  given-names: Martin
+- affiliation: Excalibur Laboratories Inc
+  family-names: Horne
+  given-names: Randy
+- affiliation: University of Texas
+  family-names: Whiteaker
+  given-names: Timothy
+- affiliation: USGS
+  family-names: Blodgett
+  given-names: David
+- affiliation: UC Irvine
+  family-names: Zender
+  given-names: Charlie
+- affiliation: EUMETSAT
+  family-names: Lee
+  given-names: Daniel
+  orcid: 0000-0002-1041-232X
+- affiliation: NCAS and University of Reading
+  family-names: Hassell
+  given-names: David
+  orcid: 0000-0001-5106-7502
+- affiliation: Corteva Agriscience
+  family-names: Snow
+  given-names: Alan D.
+- affiliation: Max Planck Institute for Meteorology
+  family-names: "K\xF6lling"
+  given-names: Tobias
+- affiliation: NOAA/CIRES/University of Colorado
+  family-names: Allured
+  given-names: Dave
+- affiliation: HDF Group
+  family-names: Jelenak
+  given-names: Aleksandar
+- affiliation: EUMETSAT
+  family-names: Soerensen
+  given-names: Anders Meier
+- affiliation: OceanDataLab
+  family-names: Gaultier
+  given-names: Lucile
+- affiliation: OceanDataLab
+  family-names: "Herl\xE9dan"
+  given-names: Sylvain
+- affiliation: Puertos del Estado, Madrid
+  family-names: Manzano
+  given-names: Fernando
+- affiliation: SMHI Rossby Centre, Swedish Meteorological and Hydrological Institute
```

Comment:
Please change my affiliation to read `SMHI`.

Reply:
Done!, the .zenodo.json already include the same affiliation string

Reply:
Sorry I was unclear, what I meant was to delete (here and in the .zenodo.json file) 
> affiliation: SMHI ~~Rossby Centre, Swedish Meteorological and Hydrological Institute~~

Reply:
Fixed